text_generation/causal_lm/cpp/generate_pipeline/README.md

```diff
@@ -0,0 +1,141 @@
+# OpenVINO Generate API
+
+## Usage 
+
+Firs of all you need to convert your model with optimum-cli
+``` sh
+optimum-cli export openvino --model "TinyLlama/TinyLlama-1.1B-Chat-v1.0" --weight-format fp16 --trust-remote-code "TinyLlama-1.1B-Chat-v1.0"
+pip install openvino-genai
+```
+
+LLMPipeline is the main object used for decoding. You can initiliza it straigh away from the folder with the converted model. It will automanically load the main model, tokenizer, detokenizer and default generation configuration.
+
+### In Python
+
+A minimalist example:
+```python
+import py_generate_pipeline as genai # set more friendly module name
```

Comment:
If you feel the name isn't friendly enough, rename the module itself instead of introducing an alias.